### ISSUE TITLE
TypeScript support

### ISSUE THREAD

**Issue by jkostrhun** [Ziv-Barber/officegen#368]
Will be good to add types for TypeScript support.

**Comment by Ziv-Barber** [comment]
Yes, I'm working on the new officegen module and it's written in TypeScript.
It can do both create office documents and also parsing existing documents.
It also working inside the browser.
Expected preview version: next month.

**Comment by noteScript** [comment]
When to support Deno?
